dbt-adapters/src/dbt/include/global_project/macros/materializations/tests/test.sql

```diff
@@ -56,7 +56,7 @@
 
   {% call statement('main', fetch_result=True) -%}
 
-    {{ get_test_sql(main_sql, fail_calc, warn_if, error_if)}}
+    {{ get_test_sql(main_sql, fail_calc, warn_if, error_if, limit)}}
```

Comment:
https://github.com/dbt-labs/dbt-adapters/pull/376 explicitly stopped passing in the limit parameter because we do **not** want to apply the limit twice when failures are stored (just once!).

So I think we'd want something like this instead:

```suggestion
    {{ get_test_sql(main_sql, fail_calc, warn_if, error_if, limit=none)}}
```